src/KubernetesClient.Aot/KubernetesClient.Aot.csproj

```diff
@@ -0,0 +1,111 @@
+    <Compile Include="..\KubernetesClient\Autorest\HttpOperationException.cs" />
+    <Compile Include="..\KubernetesClient\Autorest\HttpOperationResponse.cs" />
+    <Compile Include="..\KubernetesClient\Autorest\HttpRequestMessageWrapper.cs" />
+    <Compile Include="..\KubernetesClient\Autorest\HttpResponseMessageWrapper.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\KubernetesClient\CertUtils.cs" />
+
+    <Compile Include="..\KubernetesClient\FileSystem.cs" />
+    <Compile Include="..\KubernetesClient\IKubernetes.cs" />
+    <Compile Include="..\KubernetesClient\Kubernetes.ConfigInit.cs" />
+    <Compile Include="..\KubernetesClient\Kubernetes.cs" />
+    <Compile Include="..\KubernetesClient\KubernetesClientConfiguration.InCluster.cs" />
+    <Compile Include="..\KubernetesClient\KubernetesClientConfiguration.cs" />
+    <Compile Include="..\KubernetesClient\KubernetesException.cs" />
+
+    <Compile Include="..\KubernetesClient\ChannelIndex.cs" />
+    <Compile Include="..\KubernetesClient\IStreamDemuxer.cs" />
+    <Compile Include="..\KubernetesClient\ByteBuffer.cs" />
+    <Compile Include="..\KubernetesClient\StreamDemuxer.cs" />
+    <Compile Include="..\KubernetesClient\MuxedStream.cs" />
+    <Compile Include="..\KubernetesClient\StreamType.cs" />
+    <Compile Include="..\KubernetesClient\IKubernetes.WebSocket.cs" />
+    <Compile Include="..\KubernetesClient\Kubernetes.WebSocket.cs" />
+    <Compile Include="..\KubernetesClient\WebSocketBuilder.cs" />
+    <Compile Include="..\KubernetesClient\WebSocketProtocol.cs" />
+    <Compile Include="..\KubernetesClient\Utilities.cs" />
+    <Compile Include="..\KubernetesClient\ExecAsyncCallback.cs" />
+    <Compile Include="..\KubernetesClient\IKubernetes.Exec.cs" />
+    <Compile Include="..\KubernetesClient\Kubernetes.Exec.cs" />
+
+    <!-- <Compile Include="..\KubernetesClient\Watcher.cs" /> -->
+    <!-- <Compile Include="..\KubernetesClient\WatcherExt.cs" /> -->
+    <Compile Include="..\KubernetesClient\LineSeparatedHttpContent.cs" />
+
+    <Compile Include="..\KubernetesClient\Exceptions\KubeConfigException.cs" />
+    <Compile Include="..\KubernetesClient\Exceptions\KubernetesClientException.cs" />
+
+    <Compile Include="..\KubernetesClient\Authentication\ExecTokenProvider.cs" />
+    <Compile Include="..\KubernetesClient\Authentication\TokenFileAuth.cs" />
+    <Compile Include="..\KubernetesClient\Authentication\BasicAuthenticationCredentials.cs" />
+    <Compile Include="..\KubernetesClient\Authentication\ITokenProvider.cs" />
+    <Compile Include="..\KubernetesClient\Authentication\ServiceClientCredentials.cs" />
+    <Compile Include="..\KubernetesClient\Authentication\StringTokenProvider.cs" />
+    <Compile Include="..\KubernetesClient\Authentication\TokenCredentials.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Net.Http" Condition="'$(TargetFramework)' == 'net48'" />
```

Comment:
Seemed this can be removed since we target for `net8.0` only

Reply:
thanks, copy paste issue, will fix it